examples/pip_requirements/pip_requirements.py

```diff
@@ -34,19 +45,22 @@ def main():
         mlflow.xgboost.log_model(model, artifact_path)
         pip_reqs = get_pip_requirements(run_id, artifact_path)
         assert pip_reqs == ["mlflow", xgb_req], pip_reqs
+        print("Model URI:", mlflow.get_artifact_uri(artifact_path))
```

Comment:
Added these print statements to quickly jump to the artifact directory and make it easier to test `mlflow models serve --model-uri <model uri>`  using the logged model to make sure we can successfully restore the model.

Reply:
This isn't exactly a model URI. Can we call it an environment artifact URI? Before we print each of these, can we add another print statement that gives users some context (e.g. "Generating default environment", "Generating environment with custom pip requirements", etc...)

Reply:
Sure!